src/Compilers/CSharp/Portable/BoundTree/UnboundLambda.cs

```diff
@@ -2,13 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
```

Comment:
In the future, please consider `#nullable enable`ing a file with a commit dedicated to fixing up annotations, and then doing future work after. This is a pretty big file, and it's hard to separate the pure nullable changes from the functional changes.

Reply:
> please consider `#nullable enable`ing a file with a commit dedicated to fixing up annotations

Will do, at least for files with more than a few annotations.